cmd/vic-machine/configure/configure.go

```diff
@@ -87,7 +88,7 @@ func (c *Configure) Flags() []cli.Flag {
 	}
 
 	proxies := c.proxies.ProxyFlags(false)
-
+	dns := c.dns.DNSFlags(true)
```

Comment:
might add a comment here to mention the new issue, and after that is fixed, it should be reverted back to false.